libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -115,8 +115,8 @@ class AP_ExternalAHRS {
     bool get_speed_down(float &speedD);
     bool pre_arm_check(char *failure_msg, uint8_t failure_msg_len) const;
     void get_filter_status(nav_filter_status &status) const;
-    Vector3f get_gyro(void);
-    Vector3f get_accel(void);
+    bool get_gyro(Vector3f &gyro);
+    bool get_accel(Vector3f &accel);
```

Comment:
```suggestion
    bool get_gyro(Vector3f &gyro) WARN_IF_UNUSED;
    bool get_accel(Vector3f &accel) WARN_IF_UNUSED;
```

Force consumers to check the error state because otherwise they read garbage data.